### PR TITLE
Use correct api params for NewDot>OldDot links

### PR DESCRIPTION
**Pull request by marcaaron**
### Details
Looks like we changed the `GetShortLivedAuthToken` command to just return `shortLivedAuthToken` and `encryptedAuthToken` but didn't consider the change for NewDot > OldDot links.

### Fixed Issues
https://github.com/Expensify/App/pull/5359#issuecomment-926071634

### Tests
To test this I just quickly applied this diff and then tapped on manage cards (I don't have this set up on dev but I think it will work fine). Without the other changes we get the `undefined` stuff.

```diff
diff --git a/src/pages/workspace/WorkspaceCardPage.js b/src/pages/workspace/WorkspaceCardPage.js
index 4eee5d7f9..386c5a66e 100644
--- a/src/pages/workspace/WorkspaceCardPage.js
+++ b/src/pages/workspace/WorkspaceCardPage.js
@@ -100,13 +100,13 @@ const WorkspaceCardPage = ({
     }

     const onPress = () => {
-        if (user.isFromPublicDomain) {
-            openSignedInLink(CONST.ADD_SECONDARY_LOGIN_URL);
-        } else if (user.isUsingExpensifyCard) {
+        // if (user.isFromPublicDomain) {
+        //     openSignedInLink(CONST.ADD_SECONDARY_LOGIN_URL);
+        // } else if (user.isUsingExpensifyCard) {
             openSignedInLink(CONST.MANAGE_CARDS_URL);
-        } else {
-            openBankSetupModal();
-        }
+        // } else {
+        //     openBankSetupModal();
+        // }
     };
```

### QA Steps
Repeat steps in https://github.com/Expensify/App/pull/5359#issuecomment-926071634

### Tested On

- [x] Web
- [ ] Mobile Web
- [ ] Desktop
- [ ] iOS
- [ ] Android

### CODE REVIEW

**Review by cead22 — DISMISSED** [review]
I reproduced without this fix, and couldn't reproduce with this fix

**Review by Jag96 — DISMISSED** [review]
LGTM, good find!

File: src/libs/actions/App.js [hunk 1]
```diff
@@ -48,8 +50,8 @@ function setLocale(locale) {
  * @param {string} url relative URL starting with `/` to open in expensify.com
  */
 function openSignedInLink(url = '') {
-    API.GetShortLivedAuthToken().then((response) => {
-        Linking.openURL(`${CONFIG.EXPENSIFY.URL_EXPENSIFY_COM}${url}${url.indexOf('?') === -1 ? '?' : '&'}authToken=${response.authToken}&email=${encodeURIComponent(response.email)}`);
+    API.GetShortLivedAuthToken().then(({shortLivedAuthToken}) => {
+        Linking.openURL(`${CONFIG.EXPENSIFY.URL_EXPENSIFY_COM}${url}${url.indexOf('?') === -1 ? '?' : '&'}authToken=${shortLivedAuthToken}&email=${currentUserEmail}`);
```
Comment (iwiznia) [inline src/libs/actions/App.js:54]: We don't need to `encodeURIComponent` the email anymore?

Comment (marcaaron) [reply src/libs/actions/App.js:54]: Why not? I thought a `+` in a url mean that it's a `SPACE` and would mess up query params?

Comment (Jag96) [reply src/libs/actions/App.js:54]: With the encodeURIComponent, it doesn't log me in for an email like `joe+test@expensifail.com`, with the current code the `+` emails work for me.

Comment (marcaaron) [reply src/libs/actions/App.js]: Sorry this was my bad I moved it back so hopefully less confusing now.

**Review by Jag96 — APPROVED** [review]
LGTM

**Comment by Jag96** [comment]
Skipping unnecessary e2e to get this to staging

**Pull request merged**